dev/Projection/Microsoft.UI.Xaml.Projection.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+   <PropertyGroup>
+      <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
+      <ImplicitUsings>enable</ImplicitUsings>
+      <Nullable>enable</Nullable>
```

Comment:
We should remove these, especially `Nullable`. WinRT doesn't support annotations.